pkg/api/jobs.go

```diff
@@ -295,6 +296,41 @@ func PrintJobsReportFromDB(w http.ResponseWriter, req *http.Request,
 	RespondWithJSON(http.StatusOK, w, jobsResult)
 }
 
+func JobReportsFromDB(dbc *db.DB, release, period string) ([]apitype.Job, error) {
+
+	var start time.Time
+	var boundary time.Time
+	var end time.Time
+
+	var filterOpts = filter.FilterOptions{}
+	filterD := filter.Filter{}
+	filterOpts.Filter = &filterD
+	filterOpts.SortField = currentPassPercentage
+	filterOpts.Sort = apitype.SortDescending
```

Comment:
No need to sort here as we're just publishing so prom can scrape, is the filter required for any reason or can we skip it and save a few cpu cycles.

Reply:
I actually had to add the sort to get the scan to return results

Reply:
Ah thats a little odd, probably a bug in there for another day. Thanks.